tgui/.eslintignore

```diff
@@ -1,6 +1 @@
 /**/node_modules
-/**/*.bundle.*
```

Comment:
One advantage of moving the built files out of `tgui`, no need to exclude them from various things any more!

Reply:
😸